tools/zbfakebeacon

```diff
@@ -54,13 +54,15 @@ if __name__ == '__main__':
     parser.add_argument('-e', '--epanid', action='store', required=True, type=tohex)
     parser.add_argument('-s', '--coordinator', action='store', required=False, default=0x0000, type=tohex)
     parser.add_argument('-g', '--go', action='store_true', required=False, default=False, help="Spam Beacons, don't wait for requests")
+    parser.add_argument('-r', '--rate', action='store', required=False, default=100, help="Packets per second to spam" type=int)
```

Comment:
Missing comma between help and type assignments

Reply:
I've updated in KB repo branch of this.